diff --git a/Sources/Display.php b/Sources/Display.php
index 17bff1b..aaff2ea 100644
--- a/Sources/Display.php
+++ b/Sources/Display.php
@@ -162,7 +162,7 @@ function Display()
 	}
 
 	// Add 1 to the number of views of this topic.
-	if (empty($_SESSION['last_read_topic']) || $_SESSION['last_read_topic'] != $topic)
+	if ((empty($_SESSION['last_read_topic']) || $_SESSION['last_read_topic'] != $topic) && !$user_info['is_guest'])
 	{
 		$smcFunc['db_query']('', '
 			UPDATE {db_prefix}topics
